includes/embeds/class-amp-facebook-embed.php

```diff
@@ -14,12 +14,26 @@ class AMP_Facebook_Embed_Handler extends AMP_Base_Embed_Handler {
 	protected $DEFAULT_WIDTH = 600;
 	protected $DEFAULT_HEIGHT = 400;
 
+	/**
+	 * Tag.
+	 *
+	 * @var string embed HTML blockquote tag to identify and replace with AMP version.
+	 */
+	protected $sanitize_tag = 'div';
```

Comment:
I am curious to find out what the performance impacts are between `$this->dom->getElementsByTagName( 'div' )` and then filtering out the ones that have the required attributes (`is_facebook_raw_embed`) .vs using XPath to obtain just the exact matching set up front, for example via:

```
//div[ @data-href and ( contains( @class, 'fb-post' ) or contains( @class, 'fb-video' ) ) ]
```

Given that `div` is a very generic element, I worry about the performance impacts of locating elements in large documents. But I'm not sure if XPath would be faster.

Reply:
I created a quick benchmark checking both implementation https://gist.github.com/juanchaur1/da70708c47dadcda8e3f49c6abc5d8e0
seems like xpath isn't faster.